islandora.api.php

```diff
@@ -623,7 +623,12 @@ function hook_islandora_overview_object_alter(AbstractObject &$object, &$output)
  * Content model specific.
  */
 function hook_cmodel_pid_islandora_overview_object_alter(AbstractObject &$object, &$output) {
-  $output = $output . drupal_render(drupal_get_form('some_form', $object));
+  $view = views_embed_view('usage_collection');
+  $form['islandora_manage_overview_object'] = array(
+    '#type' => 'item',
+    '#markup' => $view,
+  );
+  $output = $output . $form . drupal_render(drupal_get_form('some_form', $object)));
```

Comment:
This... probably shouldn't be here... something of an overly-detailed example? This code doesn't actually get run, right?